crates/sui-graphql-client/src/lib.rs

```diff
@@ -551,6 +559,7 @@ impl Client {
 #[cfg(test)]
 mod tests {
     use crate::{Client, DEFAULT_LOCAL_HOST, DEVNET_HOST, MAINNET_HOST, TESTNET_HOST};
+    const NETWORKS: [&str; 3] = [MAINNET_HOST, TESTNET_HOST, DEVNET_HOST];
```

Comment:
I'd probably say we should only test against Mainnet and testnet. devnet isn't intended to be reliable enough to warrant blocking ci on